### PR TITLE
Fixes  #3341 "Page does not exist" near some usernames in Explore

### LINKED ISSUE #3341: "Page does not exist" near some usernames in Explore
1. Go to Explore
2. Scroll until you see a username with "page does not exist" appended. Example:
Screenshot_20200123-202312_Commons
Expected: Just the username.

First figure out where this string comes from (server-side? app side?) then make it not be generated, or if impossible just filter it out.

### PR DESCRIPTION
**Pull request by 318anushka**
**Description (required)**
Page does not exist issue fixed

Fixes #3341 

What changes did you make and why?

- introduced new field "user" in Media.

- displaying userName in uploadedBy.

- modified HTML parsing in getArtist.

- Replaced creator with author.

- removed getAuthorName.

- fixed delete helper since we no longer need to remove page does not exist from username.

**Tests performed (required)**

Tested ProdDebug on Oneplus 5 with API level 28.

**Screenshots (for UI changes only)**

### CODE REVIEW

**Comment by 318anushka** [comment]
I'll fix deletion in deleteHelper in a separate PR.
Using username is really great, scrolled about 100 of images and not a single one without uploadedBy.
Also i have included all the changes, please let me know if anything is left :-)

File: app/src/main/java/fr/free/nrw/commons/auth/SessionManager.java [hunk 1]
```diff
@@ -99,17 +99,6 @@ public String getUserName() {
         return account == null ? null : account.name;
     }
 
-    @Nullable
-    private String getRawUserName() {
-        Account account = getCurrentAccount();
-        return account == null ? null : accountManager().getUserData(account, KEY_RAWUSERNAME);
```
Comment (macgills) [inline app/src/main/java/fr/free/nrw/commons/auth/SessionManager.java:105]: Delete the constant `KEY_RAWUSERNAME` as well please

**Comment by neslihanturan** [comment]
Hey @318anushka , what is the status of this PR?

**Comment by sivaraam** [comment]
This has been dormant for a long time and it looks like @318anushka has not active for some time now. Should we just let interested people to take this up and get this merged?

**Comment by madisoncfallin** [comment]
I can pick it up @sivaraam!

**Pull request closed without merging**